collections/Canon_Pat_Lat/MS_Canon_Pat_Lat_172.xml

```diff
@@ -41,78 +40,217 @@
                      <author key="person_95155322">Origen</author>
-                     <title key="work_3397">Homilies on Numbers</title>
+                     <title key="work_3397">Homilies on Numbers</title><note>(tr. <persName role="trl" key="person_77679446">Rufinus</persName>)</note>
+                        <msItem>
+                           <locus>(fol. 1r–v)</locus>
+                           <note> Rufinus’s Preface</note>
+                           <rubric>Incipit prefatio … Adamanti in librum Numerorum</rubric><note> (The original name erased, replaced in brown ink with ‘Hieronimi(?)’, and subsequently overwritten with ‘Origenis’)</note>
+                           <incipit>Ut verbis tibi frater beati martyris loquar bene admontes donmate karissime</incipit>
+                           <explicit>operis nostri in profectu legentium ponat</explicit>
+                           <bibl>ed. W. A. Baehrens, <title>Origenes Werke, 7: Homilien zum Hexateuch in Rufins Übersetzung, 2: Die Homilien zu Numeri, Josua und Judices</title>, Die griechischen christlichen Schriftsteller der ersten Jahrhunderte, 30 (Leipzig, 1921), pp. 1–2.</bibl><bibl>Stegmüller, RB, no. 6178</bibl>
+                        </msItem>
+                     <msItem>
+                        <locus>(fols. 1v–2v)</locus>
+                        <note> Capitula list</note>
+                        <rubric>Incipiunt capitula. Primum</rubric>
+                        <incipit>De principio numerorum</incipit>
+                        <explicit>xxvi. Quę sit terrę sanctę descriptio terminorumque eius &amp; finium quod describit dominus</explicit>
+                        <finalRubric>Expliciunt capitula</finalRubric>
+                        <note>Only 26 numbered items due to the fact that one title is skipped between ‘xvii De prophetia eiusdem [Balaam] tercia’ and ‘xviii De visione .v. Balaam’.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 2v–170r)</locus>
+                        <note> Homilies 1–26</note>
+                        <rubric>Incipit expositio Origenis in librum Numeri </rubric>
+                        <incipit>Divinis numeris non omnes digni sunt</incipit>
+                        <explicit>Ad cuius nos hereditatem perducere dignetur dux et dominus noster Ihesus Christus cui est gloria in secula seculorum amen</explicit>
+                        <note>The first 12 homilies with added running headers ‘om prima’, ‘o ii’, ‘o iii’ etc., ‘o. xxiiii’.</note>
+                        <note>Homily 8 ends on a half-sheet, blank on the verso; Homily 9 begins on a new leaf (fol. 38r).</note>
+                        <note>Baehrens, op. cit. 3–255; Stegmüller, ibid.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 170v)</locus>
+                        <title type="desc" key="work_14541"> (Added) Noted sequence for St. Nicolas of Myrna</title>
```

Comment:
The work ID here is for 'Sequence (liturgical)', identifying only four other manuscripts. I am somewhat inclined to try keeping genre identification separate. In this case we could link to <http://www.cantusindex.org/id/ah53197> as an external identifier.